### DISCUSSION TITLE
Azure SQL support?

### DISCUSSION THREAD

**Discussion by olijaun** [apache/pekko-persistence-jdbc#228] [Q&A]
Hi

My company is migrating from on-prem to Azure. Up to now we used Oracle DB but in Azure we have to use Azure SQL.

I have no experience with Azure SQL yet. I only heard that DDL-wise etc. it should be close to MS SQL. Has anyone tried to use pekko-peristence-jdbc with Azure SQL?

Oliver

**Comment by pjfanning** [comment]
I think you would be best to contact Microsoft Support.

**Reply by olijaun** [reply]
Yes, sure, regarding the compatibility. I just wondered if anyone already did it. But yes. I will do that. Thanks!

**Comment by Roiocam** [comment]
seems like Azure SQL is based on SQL Server(Aka MSSQL), did you try SQLServer driver for the AzureSQL connecting?

**Reply by olijaun** [reply]
Thanks. No I haven't tried anything yet. I just wanted to know if anyone has already tried it and if there are some already known issues.

I will make a test setup and I will report my experience here as soon as I have some time.

**Reply by olijaun** [reply]
Update: If tried now first with Akka (2.6) and it seems to work with Azure SQL. I will now try to switch to Pekko. I will post updates here.
